overlay.d/05core/usr/lib/dracut/modules.d/20live/coreos-livepxe-rootfs.sh

```diff
@@ -32,10 +32,11 @@ elif [[ -n "${rootfs_url}" ]]; then
     fi
     # We don't need to verify TLS certificates because we're checking the
     # image hash.
+    # bsdtar can read cpio archives and we already depend on it for
+    # coreos-liveiso-persist-osmet.service.
```

Comment:
Optional. Feel free to resolve/close with no questions asked.
```suggestion
    # Note, bsdtar can read cpio archives and we already depend on it for
    # coreos-liveiso-persist-osmet.service so use it instead of cpio.
```